# The header provides a time stamp and a reference frame (frame_id)
Header header
# This provides x, y and theta
geometry_msgs/Pose2D pose2D
